RadixWallet/Features/PreAuthorizationReview/PollingStatus/PollingStatus+View.swift

```diff
@@ -0,0 +1,121 @@
+// MARK: - PreAuthorizationReview.PollingStatus.View
+extension PreAuthorizationReview.PollingStatus {
+	@MainActor
+	struct View: SwiftUI.View {
+		let store: StoreOf<PreAuthorizationReview.PollingStatus>
+
+		@ScaledMetric private var height: CGFloat = 485
```

Comment:
🤯 `ScaledMetric` ... I had completely missed this :D even though it has been around for some time. Cool!